src/svgs/icons/clipboard-icon.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
no need to specify width and height on all the tags. The view box will take care of it and it'll allow us to scale the SVG when we use it

Reply:
seems to work now: 

<img width="345" alt="screen shot 2019-02-06 at 11 10 20 am" src="https://user-images.githubusercontent.com/16960054/52367008-e365a700-29ff-11e9-80f2-62eb31342ada.png">

removed mask and g tags and other misc seemingly unnecessary info.